ocaml/xapi/xapi_vbd_helpers.ml

```diff
@@ -138,14 +138,16 @@ let valid_operations ~expensive_sharing_checks ~__context record _ref' : table =
     let fallback () =
       match Xapi_pv_driver_version.make_error_opt (Xapi_pv_driver_version.of_guest_metrics vm_gmr) vm with
       | Some(code, params) -> set_errors code params plug_ops
-      | None -> () in
-
+      | None -> ()
+    in
     (match vm_gmr with
      | None -> fallback ()
      | Some gmr ->
        (match gmr.Db_actions.vM_guest_metrics_can_use_hotplug_vbd with
         | `yes -> () (* Drivers have made an explicit claim of support. *)
         | `no -> set_errors Api_errors.operation_not_allowed ["VM states it does not support VBD hotplug."] plug_ops
+           (* according to xen docs PV drivers are enough for this to be possible *)
```

Comment:
Yes, Windows guests do not have PV drivers and `xenopsd` detects this correctly (the field is `false` when using Windows guests without the Guest Tools). They appear as detected only after the Guest Tools are installed.